common/autoware_universe_utils/src/geometry/random_convex_polygon.cpp

```diff
@@ -31,8 +31,11 @@ struct VectorsWithMin
 };
 
 VectorsWithMin prepare_coordinate_vectors(
-  const size_t nb_vertices, std::uniform_real_distribution<double> & random_double,
-  std::uniform_int_distribution<int> & random_bool, std::default_random_engine & random_engine)
+  const size_t nb_vertices,
+  std::uniform_real_distribution<double> &
+    random_double,                                   // cppcheck-suppress constParameterReference
```

Comment:
@kobayu858 
Could you put them into one line?

Reply:
@veqcc 
This line is formatted in the `clang-format -i common/autoware_universe_utils/src/geometry/random_convex_polygon.cpp` , so if it were a single line, it would fail in CI's pre-commit format.

Reply:
make sense